raiden_installer/web.py

```diff
@@ -121,6 +121,11 @@ def _send_summary(self, text, **kw):
             message["icon"] = icon
         self.write_message(message)
 
+    def _send_txhash_message(self, text, **kw):
```

Comment:
Just a nitpick, if you already go that way and build a function to send a message with tx_hash property, why do you go the way of passing it through `kw`
Couldnt you just create an tx_hash argument?

Reply:
Yes thats also a good point, if we need something more then probably we can pass it later as well. Right now just keeping it `tx_hash` will increase the readable.